postprocess/patch-exoscale.jsonnet

```diff
@@ -0,0 +1,31 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+
+// The hiera parameters for the component
+local params = com.inventory().parameters.cert_manager;
+
+local pki_file = std.extVar('output_path') + '/pki.yaml';
+local pki_obj = com.yaml_load_all(pki_file);
+
+local deploy_file = std.extVar('output_path') + '/deployment.yaml';
+local deploy_obj = com.yaml_load(deploy_file);
+
+{
+  [if params.components.exoscale_webhook.enabled then 'pki']: [
+    if obj.kind == 'Certificate' then obj {
+      spec+: {
+        duration: '%s0m0s' % obj.spec.duration,
```

Comment:
why is this necessary? According to the documentation, cert-manager only requires that the value is accepted by Go's `time.ParseDuration()` (https://golang.org/pkg/time/#ParseDuration) which is fine with just `1000h` and doesn't need the explicit '0m0s'.

Reply:
I run into this issue with TopoLVM, where it was expanded to that format and ArgoCD complained.